var/spack/repos/builtin/packages/charliecloud/package.py

```diff
@@ -19,7 +19,12 @@ class Charliecloud(AutotoolsPackage):
     license("Apache-2.0")
 
     version("master", branch="master")
-    version("0.35", sha256="042f5be5ed8eda95f45230b4647510780142a50adb4e748be57e8dd8926b310e")
+    version("0.36", sha256="b6b1a085d8ff82abc6d625ab990af3925c84fa08ec837828b383f329bd0b8e72")
+    version(
+        "0.35",
+        deprecated=True,
+        sha256="042f5be5ed8eda95f45230b4647510780142a50adb4e748be57e8dd8926b310e",
+    )
```

Comment:
Is there a security reason to deprecate the old version so quickly? We generally prefer to allow multiple versions otherwise.

Reply:
No security concern; rather, we just don't support anything aside from the latest release.

That said, I have bumped into issues where concretization fails when specifying one of these older versions for local testing. I find myself commenting out the deprecated flag instead of figuring out how to ignore it during concretization. So perhaps I will just keep a handful alive at time.